packages/angular/src/editor/link-dialog.component.ts

```diff
@@ -110,12 +110,20 @@ export class LinkDialogComponent extends ResponsiveComponent implements OnInit,
     // Event Settings
     editor: TriggerInterface
     eventManager: TriggerInterface
+    element: Element
     eventInsert = true
 
     // UI Settings
     isListView = false
     selected: Array<number> = []
 
+    // User Input
+    content: ContentEntity
+    linkType: string = ''
+    linkText: string = ''
```

Comment:
travis is complaining about these being typed as a string/boolean a second time (as the set/= already does so)